dojo/notifications/helper.py

```diff
@@ -61,7 +61,8 @@ def create_notification(event=None, *args, **kwargs):
 
         # only send to authorized users or admin/superusers
         if product:
-            users = users.filter(Q(id__in=product.authorized_users.all()) | Q(is_superuser=True) | Q(is_staff=True))
+            qs = (product.authorized_users.all() | product.prod_type.authorized_users.all()).distinct()
+            users = users.filter(Q(id__in=qs) | Q(is_superuser=True) | Q(is_staff=True))
```

Comment:
now I'm curious because I would expect this would work just as well:
`users = users.filter(Q(id__in=product.authorized_users.all()) | Q(id__in=productprod_type.authorized_users.all()) | Q(is_superuser=True) | Q(is_staff=True))`